client/components/site-selector/index.jsx

```diff
@@ -73,17 +73,19 @@ class SiteSelector extends Component {
 		groups: false,
 	};
 
-	state = {
+	static initialState = {
 		highlightedIndex: -1,
 		showSearch: false,
 		isKeyboardEngaged: false,
 	};
 
+	state = this.constructor.initialState;
+
 	reset() {
```

Comment:
I was trying to trigger this to confirm correct behavior, but it appears to be unused. Can you find whether this method is referenced anywhere? We might be able to simplify this further and not need to make the `initialState` changes.

Looks like this was introduced in #5808 by @marekhrabe (8b08a8f2ad53d33e65fe48970b6114bd121f9a61).

Can you say whether it's safe to remove this method which doesn't appear to be called?

Reply:
Reset is not in use right now. It was part of work for #8739 that wasn't implemented. I think it's safe to remove now and simplify the initialState declaration. It can be easily reintroduced once we are sure we need it.